web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/ResourceLinkFactory.java

```diff
@@ -25,6 +25,10 @@
 
 public class ResourceLinkFactory {
 
+    private ResourceLinkFactory() {
+
```

Comment:
Maybe add ``` throw new IllegalStateException("Utility class");```

Reply:
I don't think it's necessary since the constructor is private but I can add it